src/xenia/kernel/xboxkrnl/xboxkrnl_module.cc

```diff
@@ -189,6 +189,14 @@ XboxkrnlModule::XboxkrnlModule(Emulator* emulator, KernelState* kernel_state)
                                        ordinals::XexExecutableModuleHandle,
                                        ppXexExecutableModuleHandle);
 
+  // ExLoadedImageName (char*)
+  // The full path to loaded image/xex including its name.
+  // Used usually in custom dashboards (Aurora)
+  uint32_t ppExLoadedImageName =
+      memory_->SystemHeapAlloc(kExLoadedImageNameSize);
```

Comment:
Minor nitpick: initialize this to zeros (`std::memset(...)`), I don't think `SystemHeapAlloc` guarantees a zeroed buffer?